### DISCUSSION TITLE
FastestVPN

### DISCUSSION THREAD

**Discussion by twillist** [qdm12/gluetun#1214] [Q&A] [answered]
I'm having trouble getting gluetun to work with FastestVPN. It tries to connect then fails. When I try to update the server list with the command in the wiki it goes through all the files with the error "filename does not have a protocol suffix" then says "not enough servers found: 0 and expected at least 52." Is FastestVPN still supported?

**Comment by qdm12** [comment, marked as answer]
Thanks for reporting this :+1: 

Fastestvpn changed their downloadable OpenVPN filenames to end with `tcp.ovpn` and `udp.ovpn` where it was before `TCP.ovpn` and `UDP.ovpn` so my code wouldn't work :cry: e7d5ae5dc183af6c1f75af31128f4166bdf72ba7 changes it to be case insensitive, and 63890c159ed5f3757aa5afd97044f34d10736ca2 updates the fastestvpn servers data built-in the image. You can probably pull the latest image and get this solved :wink:
